### PR TITLE
[jsk_fetch_startup] add a fetch head box link to xacro

### PR DESCRIPTION
**Pull request by sktometometo**

### CODE REVIEW

File: jsk_fetch_robot/jsk_fetch_startup/robots/jsk_fetch.urdf.xacro [hunk 1]
```diff
@@ -37,4 +38,32 @@
...
+  <link name="head_box_link" >
+    <inertial>
+      <origin rpy="0 0 0" xyz="0 0 0" />
+      <mass value="1" />
+      <inertial ixx="1" ixy="0" ixz="0" iyy="1" iyz="0" izz="1" />
+    </inertial>
+    <visual>
+      <origin rpy="0 0 0" xyz="0 0 0" />
+      <geometry>
+        <mesh filename="package://jsk_fetch_accessories/head_box/fetch_head_box_simplified.stl" scale="0.001 0.001 0.001" />
```
Comment (knorth55) [inline jsk_fetch_robot/jsk_fetch_startup/robots/jsk_fetch.urdf.xacro:56]: i can't find this stl file, but where is this stl file?

**Review by knorth55 — CHANGES_REQUESTED** [review]

File: jsk_fetch_robot/jsk_fetch_startup/robots/jsk_fetch.urdf.xacro (hunk 1 above)
Comment (sktometometo) [reply jsk_fetch_robot/jsk_fetch_startup/robots/jsk_fetch.urdf.xacro:47]: I'll add the stl with another PR.

Comment (sktometometo) [reply jsk_fetch_robot/jsk_fetch_startup/robots/jsk_fetch.urdf.xacro]: Please refer https://github.com/jsk-ros-pkg/jsk_robot/pull/1249

Comment (sktometometo) [reply jsk_fetch_robot/jsk_fetch_startup/robots/jsk_fetch.urdf.xacro]: Please refer to https://github.com/jsk-ros-pkg/jsk_robot/pull/1250

**Pull request merged**